community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/planner/logical/steps/LogicalPlanProducer.scala

```diff
@@ -594,6 +594,9 @@ case class LogicalPlanProducer(cardinalityModel: CardinalityModel) extends Colle
   def planEager(inner: LogicalPlan) =
     Eager(inner)(inner.solved)
 
+  def planError(inner: LogicalPlan, exception: ShortestPathFallbackDisabledException) =
+    ErrorPlan(inner, exception)(inner.solved)
```

Comment:
I don't know why, but I find this pretty cool, having an ErrorPlan operator! :-)